api/src/main/java/org/openmrs/module/idgen/IdentifierPool.java

```diff
@@ -131,6 +133,7 @@ public void setMinPoolSize(Integer minPoolSize) {
 	/**
 	 * @return boolean describing whether this is sequential
 	 */
+	@JsonIgnore
```

Comment:
Why ignore this?

Reply:
As a matter of fact that we have #getSequential() which the standard getter, while Jackson is parsing this file into json it also evaluates this as a getter hence resolving into conflicting getters